### DISCUSSION TITLE
Configure the default Compose Template

### DISCUSSION THREAD

**Discussion by markhaines** [louislam/dockge#256] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

It would be time saving to be able to edit the default compose.yaml & associated .env which is presented when you click +Compose so you can add variables that tend to stay common across all your instances, such as path, time zone, PUID & PGID, network etc.

### ✔️ Solution

expose the default compose.yaml in the config folder and allow it to be edited.

**Comment by thutex** [comment]
i agree that it would be very nice and timesaving to have the default compose file editable, wheter it be through a file or through db doesn't really matter - as long as it can be changed, as i am much more likely to reuse the same network and timezone rather than setting up another nginx container.

**Comment by BernardoEstrada** [comment]
+1, seems like there is a pr for it #922
